vnext/Microsoft.ReactNative/Views/VirtualTextViewManager.cpp

```diff
@@ -30,10 +31,41 @@ void VirtualTextShadowNode::AddView(ShadowNode &child, int64_t index) {
   if (auto span = childNode.GetView().try_as<xaml::Documents::Span>()) {
     auto &childVTSN = static_cast<VirtualTextShadowNode &>(child);
     m_highlightData.data.emplace_back(childVTSN.m_highlightData);
+    AddToPressableCount(childVTSN.m_pressableCount);
   }
   Super::AddView(child, index);
 }
 
+void VirtualTextShadowNode::onDropViewInstance() {
+  AddToPressableCount(-m_pressableCount);
+  Super::onDropViewInstance();
+}
+
+void VirtualTextShadowNode::AddToPressableCount(int count) {
+  m_pressableCount += count;
+  if (const auto uiManager = GetNativeUIManager(GetViewManager()->GetReactContext()).lock()) {
```

Comment:
just to confirm, this data doesn't need to be synchronized on, right?

Reply:
Can you elaborate? What do you mean by "synchronized on"?